BoxContentSDK/BoxContentSDK/Clients/BOXContentClient+File.h

```diff
@@ -110,6 +110,17 @@
 - (BOXFileDeleteRequest *)fileDeleteRequestWithID:(NSString *)fileID;
 
 /**
+ *  Generate a background request to delete a file.
+ *
+ *  @param fileID File ID.
+ *  @param associateID   an Id to associate with this background upload task to reconnect to if needed
```

Comment:
I find this sentence a little awkward to parse. I think we could have fairly consistent language here and below. I see there are other comments along the same lines in the code, which I also find difficult, but we don't have to audit them all for this PR.

Maybe something like `A unique ID used to recover background tasks. Providing an ID associated with a running background task will cause the request to reconnect to that task, rather than making a new one.`

@thuyBox What do you think?